assets/react/v3/shared/components/fields/FormInputWithContent.tsx

```diff
@@ -124,10 +118,9 @@ const styles = {
   `,
   input: (contentPosition: string, showVerticalBar: boolean, size: string) => css`
     ${typography.body()};
-    height: 26px;
     border: none;
     box-shadow: none;
-    padding: 0;
+    padding-inline: ${spacing[12]};
```

Comment:
Why do we need an extra padding?

Reply:
Sorry, we actually don't need extra padding.